identity/id_registration_confirmation_task.go

```diff
@@ -14,94 +14,97 @@ import (
 )
 
 const (
-	IdRegistrationConfirmationTaskName string = "IdRegistrationConfirmationTaskName"
+	idRegistrationConfirmationTaskName string = "IDRegistrationConfirmationTaskName"
```

Comment:
Does it make logical sense to call the value "IDRegistrationConfirmationTaskName" as well?